selene_sdk/samplers/online_sampler.py

```diff
@@ -140,20 +133,14 @@ def __init__(self,
         np.random.seed(self.seed)
         random.seed(self.seed + 1)
 
-        if (sequence_length + center_bin_to_predict) % 2 != 0:
-            raise ValueError(
+        if isinstance(center_bin_to_predict, int):
```

Comment:
sorry I forget - is multibin still in this branch? I see a lot of references to `center_bin_to_predict` type checking, seeing if it's a window, etc. etc. in this file - basically ALL the changes to this file seem to be for multibin or something I don't remember? - but the documentation for that parameter isn't updated. if multibin isn't in this branch any longer, center_bin_to_predict is always an int right?

Reply:
no there is no multibin in this branch : ) center_bin_to_predict can be an int or specify an interval by a list of two ints (required for training Sei)

Reply:
Thanks @jzthree for all your work on this PR - I'll do a final review of this PR in the next few days. Just wanted to ask, are the Sei coordinates 'off center'? (because of even length sequence context)